fli-iam/shanoir-ng
# shanoir-issue#2199: add studycard policy to study details + save in database

How to test:
Create or update a study
Change the value of "study card policy"
Check in database that it was stored

This PR only adds a new field, it is not used yet.

This PR also fixes https://github.com/fli-iam/shanoir-ng/issues/2103
